README.md

```diff
@@ -1 +1,7 @@
-# IPE-competencies-updater
\ No newline at end of file
+# IPE-competencies-updater
+
+This Project is for running the InterProfessional Education(IPE) courses data for outputting the Learning outcomes and there by helping the IPE units and partnering units like Pharmacy, Social Work etc for data informed decision making. This process after end of term primarly for Fall/Winter/Spring-Summer terms.  
+
+### Docker Run
+1. Building the project `docker-compose build`
```

Comment:
Minor: But this command is `docker compose` now rather than `docker-compose`. https://docs.docker.com/compose/cli-command/

Reply:
OK, I changed in the Readme. I think I have my docker alias still with docker-compose so In my mind I am still stuck with that. But I feel at some point Docket will stop supporting this. I will update my alias as well

Reply:
Right, should just resolve your aliases. It'll probably work for awhile but be removed eventually since it's a command now.